src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum ESortType {
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'by length',
+  DEFAULT = '',
+}
+
+interface IOptions {
+  sortBy: ESortType;
+  isReversed: boolean;
+}
+
+const getGoods = (goods: string[], { sortBy, isReversed }: IOptions) => {
```

Comment:
We can keep all the stuff in a single props object.
It looks a bit messy when some props types are defined in the interface and some in the function directly

```suggestion
interface Options { // you have it above, added it as an example
  goods: string[];
  sortBy: SortType;
  isReversed: boolean;
}

const getGoods = ({ goods, sortBy, isReversed }: Options) => {
```